core/drivers/crypto/hisilicon/sub.mk

```diff
@@ -0,0 +1,5 @@
+cppflags$(sm) += -Icore/drivers/crypto/hisilicon/sec
+cppflags$(sm) += -Icore/drivers/crypto/hisilicon/qm
```

Comment:
Include directories are added with `incdirs-y`. I suggest `incdirs-y += include` here and have the header files under `core/drivers/crypto/hisilicon/include`.